admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -129,20 +131,19 @@ Edit raw filter instead:
   
   Example:
 
-  * *objectClass=inetOrgPerson*
+  * *(&(objectClass=inetOrgPerson)(memberOf=cn=owncloudusers,ou=groups,dc=example,dc=com))*
 
 x users found:
   This is an indicator that tells you approximately how many users will be
-  allowed to access ownCloud. The number updates automatically after any 
-  changes.
+  listed in ownCloud. The number updates automatically after any changes.
 
 Login Filter
 ~~~~~~~~~~~~
 
-The settings in the Login Filter tab determine what the user's login will be, 
-for example an LDAP username, or an email address. You may select multiple user 
-details. (You may bypass the form fields and enter a raw LDAP filter if you 
-prefer.)
+The settings in the Login Filter tab determine which LDAP users can log in to your
+ownCloud system, and how: what the user’s login will be, for example an LDAP username,
+or an email address. You may select multiple user details. (You may bypass the form fields
```

Comment:
instead of "and how: …" (seems unnatural, hard to read) i suggest "and which attribute or attributes the provided login name is matched against (e.g. LDAP/AD username, email address)."